### ISSUE TITLE
understanding the architecture/process

### ISSUE THREAD

**Issue by wdduncan** [fbelleau/superset4sparql#1]
Hi @fbelleau I appreciate your initiative in starting this project. 
I'm trying to get a better understanding of the what the architecture and related processes needs to look like.

For getting the SPARQL query results into a form that Superset can use, are wanting to use relational database as an intermediary?
For example:

<img width="428" alt="image" src="https://user-images.githubusercontent.com/3186638/229296453-f6eea90d-dc44-47d6-a84a-421ef7c91326.png">

**Comment by fbelleau** [comment]
There is no intermediate database, this would be a huge drawback. My design is based on a ES REST SQL query to SPARQL query converter done in real time by an intermediate FLASK REST service.

Here is the architecture :

<img width="889" alt="image" src="https://user-images.githubusercontent.com/995409/229835851-0e1f4836-d3ae-482f-8796-30cd70dc1f02.png">

**Comment by fbelleau** [comment]
No, I have illustrated the architecture in the ticket. Superset send Sparql
query and expose the results set.

Le sam. 1 avr. 2023 10 h 53, Bill Duncan ***@***.***> a
écrit :
